src/styles/getModalStyles.js

```diff
@@ -83,7 +84,9 @@ export default (type, windowDimensions, popoverAnchorPosition = {}, containerSty
             swipeDirection = ['down', 'right'];
             animationIn = isSmallScreenWidth ? 'slideInRight' : 'fadeIn';
             animationOut = isSmallScreenWidth ? 'slideOutRight' : 'fadeOut';
-            shouldAddTopSafeAreaPadding = true;
+
+            // Only apply top padding on iOS since only iOS using SafeAreaView and the top insets is not apply
```

Comment:
Is the comment complete here @mollfpr ?

Reply:
Yes, it’s complete

Reply:
```suggestion
            // Only apply top padding on iOS since it's the only platform using SafeAreaView
```

I think this tweak clears up the comment a little.

Reply:
Thanks for clearing it up, I’ll add this as a commit.